wallet/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -389,6 +390,10 @@ getAccountAddrMaps accId = do
   where
     getMap aiLens = fmap (fromMaybe mempty) $ preview $ wsAccountInfos . ix accId . aiLens
 
+-- | A simple way to fetch all the account @CAddress@.
+getAccountAddresses :: WebTypes.AccountId -> Query (CAddresses)
```

Comment:
This may seem silly, but the reasoning is If we ever change something with the `CAddresses` we don't have to add other parentheses. Which sounds mad when I actually write it down, I can fix this.